lib/experimental/modules/class-gutenberg-modules.php

```diff
@@ -215,4 +221,4 @@ function gutenberg_enqueue_module( $module_identifier ) {
 add_action( 'wp_head', array( 'Gutenberg_Modules', 'print_module_preloads' ) );
 
 // Prints the script that loads the import map polyfill in the footer.
-add_action( 'wp_footer', array( 'Gutenberg_Modules', 'print_import_map_polyfill' ), 11 );
+add_action( 'wp_head', array( 'Gutenberg_Modules', 'print_import_map_polyfill' ), 11 );
```

Comment:
Just curious what's the rationale from changing from `wp_footer` to `wp_head`? Is it because the other modules are printed in the head?

If so, wouldn't we want to also print it _before_ the module scripts (i.e. lower priority than `10`)?

Reply:
I mirrored WP core as there might be something specific in how `document.write` works. Once I confirm it works as expected in not supported browsers, I will play with where it should be located.

Reply:
@aduth knows best as he discovered this technique for polyfills.

Reply:
I'm merely wondering about the order of scripts. Isn't the polyfill needed _before_ the relevant `script[type="importmap"]` tags so that it works?

Reply:
The order doesn't seem to matter. I don't know how much we can move it later in the HTML document:

<img width="1460" alt="Screenshot 2023-12-20 at 23 00 26" src="https://github.com/WordPress/gutenberg/assets/699132/a5afafe4-9639-450c-a426-98603072abb7">

Reply:
I tested it again today, and it looks like it works also when included in the footer. However, I don't know how much it's related to how Interactivity API works.